modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteCacheInterceptorSelfTestSuite.java

```diff
@@ -63,12 +63,6 @@
         GridTestUtils.addTestIfNeeded(suite,GridCacheInterceptorReplicatedSelfTest.class, ignoredTests);
         GridTestUtils.addTestIfNeeded(suite,GridCacheInterceptorReplicatedWithStoreSelfTest.class, ignoredTests);
 
-// TODO GG-11141.
-//        GridTestUtils.addTestIfNeeded(suite,GridCacheOnCopyFlagTxPartitionedSelfTest.class, ignoredTests);
-//        GridTestUtils.addTestIfNeeded(suite,GridCacheOnCopyFlagReplicatedSelfTest.class, ignoredTests);
-//        GridTestUtils.addTestIfNeeded(suite,GridCacheOnCopyFlagLocalSelfTest.class, ignoredTests);
-//        GridTestUtils.addTestIfNeeded(suite,GridCacheOnCopyFlagAtomicSelfTest.class, ignoredTests);
-
         GridTestUtils.addTestIfNeeded(suite,CacheInterceptorPartitionCounterRandomOperationsTest.class, ignoredTests);
```

Comment:
I suggest to delete unused classes from code, as well

Reply:
Done

Reply:
Classes did not contain test cases and have been removed:

1. GridCacheOnCopyFlagTxPartitionedSelfTest
2. GridCacheOnCopyFlagReplicatedSelfTest
3. GridCacheOnCopyFlagLocalSelfTest
4. GridCacheOnCopyFlagAtomicSelfTest